src/index.js

```diff
@@ -53,11 +54,12 @@ ${stringifyObject(config)}
         })
     })
     .catch(parsingError => {
-      console.error(`Could not parse lint-staged config.
-Make sure you have created it. See https://github.com/okonet/lint-staged#readme.
+      console.error(dedent`
+        Could not parse lint-staged config.
+        Make sure you have created it. See https://github.com/okonet/lint-staged#readme.
```

Comment:
isn't it better if we set the link to https://github.com/okonet/lint-staged#configuration?

Reply:
or just https://github.com/okonet/lint-staged

Reply:
Configuration is better IMO. No need to search for it.